src/Compositor.cpp

```diff
@@ -1245,24 +1245,38 @@ bool CCompositor::isWindowActive(CWindow* pWindow) {
 }
 
 void CCompositor::moveWindowToTop(CWindow* pWindow) {
+    changeWindowZOrder(pWindow, true);
```

Comment:
why not rid of this? internal func, we can just yeet it and replace the refs with changewindowzorder